src/tests/conformance/cmd/endpoints.rs

```diff
@@ -0,0 +1,26 @@
+use crate::{
+    protocol::codecs::binary::{BinaryMessage, Payload},
+    tests::conformance::perform_response_test,
+    tools::config::TestConfig,
+};
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn c019_TM_ENDPOINTS_node_should_send_endpoints_after_handshake() {
+    // ZG-CONFORMANCE-019
+
+    // Check for a TmEndpoints message.
+    let check = |m: &BinaryMessage| matches!(&m.payload, Payload::TmManifests(..));
+    perform_response_test(Default::default(), &check).await;
+}
+
+#[should_panic]
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn c020_TM_ENDPOINTS_node_should_not_send_endpoints_if_no_handshake() {
```

Comment:
More of a question.
Will the synth node receive any message if we skip the handshake procedure?
If not, maybe we could simplify and put all these `do_not_receive_if_no_handshake` tests under one test?

Reply:
I have to check this, at the moment IDK.

Reply:
Will do in the next PR.